cartography/intel/gcp/iam.py

```diff
@@ -705,6 +709,57 @@ def attach_role_to_domain(
     )
 
 
+def _set_used_state_tx(
+    tx: neo4j.Transaction, project_id: str, common_job_parameters: Dict, update_tag: int,
+) -> None:
+    ingest_role_used = """
+    MATCH (n:GCPRole)<-[:RESOURCE]-(:GCPProject{id: {GCP_PROJECT_ID}})
```

Comment:
same as other clouds, let's flip this query. start with CloudanixWorkspace->.... AzureRole